src/pytorch_lightning/strategies/fully_sharded_native.py

```diff
@@ -66,6 +71,8 @@ def __init__(  # type: ignore[no-untyped-def]
         process_group_backend: Optional[str] = None,
         cpu_offload=None,
         backward_prefetch=None,
+        mixed_precision=None,
```

Comment:
we may want to consider the story around checkpointing as well - there are currently 3 different APIs and some configuration that we support, it would be good to include this in lightning as well.